internal/blueprint/blueprint_test.go

```diff
@@ -18,24 +18,16 @@ func TestDeepCopy(t *testing.T) {
 	}
 
 	bpCopy, err := bpOrig.DeepCopy()
-	if err != nil {
-		t.Fatalf("Blueprint.DeepCopy failure: %s", err.Error())
-	}
-	if diff := cmp.Diff(bpOrig, bpCopy); diff != "" {
-		t.Fatalf("Blueprint.DeepCopy is different from original\ndiff: %s", diff)
-	}
+	require.Nilf(t, err, "Blueprints.DeepCopy failure: %#v", err)
```

Comment:
An alternative to this is NoError

Reply:
I tried it like this:
```
    if require.NoError(t, err) {
        require.Equalf(t, bpOrig, bpCopy, "Blueprints.DeepCopy is different from original.")
    }
```
which should be the same as in the documentation but I'm getting this:
```
$ go test -v -race -covermode atomic -coverprofile=profile.cov .
# github.com/osbuild/osbuild-composer/internal/blueprint [github.com/osbuild/osbuild-composer/internal/blueprint.test]
./blueprint_test.go:21:23: require.NoError(t, err) used as value
FAIL    github.com/osbuild/osbuild-composer/internal/blueprint [build failed]
```
What am I doing wrong?

Reply:
Should have used assert, but I used it wrong anyway, should be better now.